README.md

```diff
@@ -410,6 +412,20 @@ _Default: 10_
 
 Length of random salt used when generating password hash.
 
+#### __Provisioning.SetHostName__
+
+_Type: Boolean_
+_Default: y_
+
+Control if WAAgent should attempt to set the VMs hostname
```

Comment:
Could you explain when this would be used? Perhaps add an example for when you would want to change from the default and *not* set the hostname.

Nit: periods at the end of the sentence for document consistency.